src/child.ts

```diff
@@ -2,8 +2,7 @@
  * Strongly typed accessors for children, since arrow.Data.children[] is untyped
  */
 
-import { Data } from "apache-arrow/data";
-import { Vector } from "apache-arrow/vector";
+import { Data, Vector } from "apache-arrow";
 import { Float } from "apache-arrow/type";
```

Comment:
Should this import be changed too?

Reply:
Ohh I thought these are just types. oki let's import all of these from `/`